centralledger/requirements.yaml

```diff
@@ -42,7 +42,7 @@ dependencies:
   alias: mysql
   condition: mysql.enabled
 - name: kafka
-  version: 0.7.2
+  version: 0.13.11
```

Comment:
Please can you also update the container versions for Kafka and Zookeeper. I am almost certain that they have also been updated, and should be kept in sync.